code/lib/test/template/stories/loader-enhancements.stories.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @typescript-eslint/naming-convention,storybook/prefer-pascal-case */
+import { expect, within, userEvent as globalUserEvent } from '@storybook/test';
+
+const meta = {
+  component: globalThis.Components.Button,
+  parameters: { chromatic: { disable: true } },
+  args: { label: 'Button' },
+};
+
+export default meta;
+
+export const canvas_is_equal_to_within_canvas_element = {
+  async play({ canvas, canvasElement }) {
+    const oldCanvas = within(canvasElement);
+    await expect(canvas satisfies typeof oldCanvas).toEqual(within(canvasElement));
+  },
+};
+
+// TODO enable this in a later PR, once we have time to QA this properly
+// export const context_user_event_is_equal_to_user_event_setup = {
+//   async play({ userEvent }) {
+//     await expect(userEvent satisfies typeof globalUserEvent).toEqual(globalUserEvent.setup());
```

Comment:
I suspect this won't be equal, as `setup()` supposedly creates a new instance of UserEvent.

Reply:
Yes, but `toEqual` doesn't check for reference. Structurally it was equal.